src/schema/objects/common_principles.yaml

```diff
@@ -3,6 +3,18 @@
 # WARNING: The terms are presented here in alphabetical order!
 # The order in which these terms are presented in the specification is defined in `rules/common_principles.yaml`,
 # rather than this file (`objects/common_principles.yaml`).
+atlas:
+  display_name: Atlas
+  description: |
+    An *atlas* is structured neuroanatomical or neurofunctional information expressed as spatial annotations,
+    such as anatomical landmarks, segmentations, parcellations, probabilistic maps, or feature maps,
+    typically defined in relation to a *space* engendered by a specific *template*.
+    Within BIDS, atlases provide standardized spatially-organized knowledge to label, localize,
+    and interpret data features.
+
+    Atlases often rely on or embed templates as their reference framework.
+    In BIDS, templates establish the spatial coordinate frames for atlases,
+    independently of whether or not they are considered part of specific atlases' outcomes.
```

Comment:
This newline breaks rendering:

![image](https://github.com/user-attachments/assets/e3ea370c-26cb-4ffa-b7e4-0e25c2a8befb)


```suggestion
    Atlases often rely on or embed templates as their reference framework.
    In BIDS, templates establish the spatial coordinate frames for atlases,
    independently of whether or not they are considered part of specific atlases' outcomes.
```